spec/parsePdr/ParsePdrSuccessSpec.js

```diff
@@ -61,8 +61,37 @@ describe("The Parse PDR workflow", function() {
       lambdaOutput = await lambdaStep.getStepOutput(workflowExecution.executionArn, "QueueGranules");
     });
 
-    it("has expected path and name output", function() {
-      expect(lambdaOutput.payload).toEqual({ granules_queued: 1 });
+    it("has expected pdr and arns output", function() {
+      expect(lambdaOutput.payload.running.length).toEqual(1);
+      expect(lambdaOutput.payload.pdr).toEqual(expectedParsePdrOutput.pdr);
+    });
+  });
+
+  describe('the PdrStatusCheck Lambda', () => {
+    let lambdaOutput = null;
+
+    beforeAll(async function() {
+      lambdaOutput = await lambdaStep.getStepOutput(workflowExecution.executionArn, "PdrStatusCheck");
```

Comment:
I noticed that lambdaStep.getStepOutput takes the lambda function name instead of step name, or it won't find the step.